docs/modules/ROOT/pages/extensions/mock.adoc

```diff
@@ -26,3 +26,37 @@ Please refer to the above link for usage and configuration details.
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Usage
+
+To use camel-mock capabilities it is required to get access to MockEndpoint instances.
+
+CDI injection could be used for accessing instances (see https://quarkus.io/guides/getting-started-testing#injection-into-tests[Quarkus documentation]).
+You can inject camelContext into test using `@Inject` annotation. Camel context can be then used for obtaining mock endpoints.
+See the following example:
+
+----
+@Inject
+CamelContext camelContext;
+
+
+@Test
+public void test() throws InterruptedException {
+
+    //test code
+
+    MockEndpoint mockEndpoint = camelContext.getEndpoint("mock:result", MockEndpoint.class);
+
+    mockEndpoint.expectedBodiesReceived("Hello World");
```

Comment:
There should be some route invocation at this location (through rest or directly over Java API). Otherwise the test does not make much sense. It would also be nice to have the route definition in the example.